src/test/java/com/ericsson/esj/jenkins_manager_test/JenkinsManagerExecutionTest.java

```diff
@@ -22,40 +38,100 @@
             Arrays.asList("Test_Job1", "Test_Job2", "Test_Job3", "Test_Job4", "Test_Job5", "Test_Job6", "Test_Job7",
                     "Test_Job8", "Test_Job9", "Test_Job10", "Test_Job11", "Test_Job12", "Test_Job13", "Test_Job14",
                     "Test_Job15", "Test_Job16", "Test_Job17", "Test_Job18", "Test_Job19", "Test_Job20"));
+    static JenkinsManager jenkinsManager;
 
-    public static void main(String[] args) throws URISyntaxException, IOException, InterruptedException {
```

Comment:
Maybe use junit or something like that for the tests instead? Feels better to use some common framework